### ISSUE TITLE
Library overview numbers should not bunch +0 games and +0 DLC together

### ISSUE THREAD

**Issue by Luckz** [bartervg/barter.vg#234]
# What problem does this feature address?
![image](https://user-images.githubusercontent.com/224748/107119104-a5691b00-6885-11eb-86ef-443d5aa1b085.png)
- The user expectation is that ⚙️ is the +0 games count. The alt text description even literally says it's **games**. IMO it should be games + apps ~~+ videos~~, since IIRC those contribute to Games Count™. Tools and Configs and such do not contribute to games I believe. **Correct me.**
- The DLC count already contains +0 DLCs. I think +1 and +0 DLCs _perhaps_ should be listed separately. Currently it looks like it would show only +1 DLCs, but that's not the case. And/or it's non-store DLC items. My Barter DLC count is almost twice of my profile DLC count. I guess we shouldn't treat +0 store listed DLC the same as non-store DLCs, so to be honest I have no idea what to do here – **input?**
- Similarly, ⚙️ and ⎚ are not separate sets but have common items. The whole thing is a big Venn diagram, but the "+" might imply that it's not. At least to me it has that implication. **Discuss **(below)**!**

- "Steam games" could be a clickable link to _non-DLC_ tagged _SteamPlus1_? https://github.com/bartervg/barter.vg/issues/231 willing that now works. [IDK if we have tools and configs and crapware like advert app IDs tagged as +1. Investigate? We also have playtests tagged as SteamPlus1, and withholding the conversation that they should be excised from Barter altogether, they definitely don't +1 anything, or do they?]

## Describe a solution
- Split ⚙️ or make it non-DLC-only.
- `Refactor` alt text 😊
- Add new link(s), if easy.
- Have a lengthy design by committee session in here about the other topics.

- _Enjoy the bonus below._

# Bonus
We could have a big, discriminatory conversation about having a total number of all Steam items connected to an account shown as 'Library'. The Library number is not Steam Library but the sum of all libraries, which might be counter-intuitive on a predominantly-Steam trading site.

Maybe **alt text** for the library number could have _Steam +1 total_, _Steam everything-standalone total_, and _Steam Barter items total_ (= packages + DLCs + tools + configs + fake crap added to the previous one).
Having an alt text means I would be able to see this information without waiting a good while to load an entire library. OTOH if that's not cached conveniently. displaying this on every user-related page would add **humongous** server load. __Tread with care!__
Do I need to make an extra issue for this feature request?

**Comment by bartervg** [comment]
Could you provide the mock up of your ideal library tab and game count details (screenshot and/or text)?

**Comment by bartervg** [comment]
- Split ⚙️ or make it non-DLC-only.
I changed the ⚙️ count to game-only. After additional consideration, I expanded it to game OR Application OR Mod. 
>  games + apps + videos

I don't believe that videos or movies grant +1.

- I think +1 and +0 DLCs perhaps should be listed separately.
I split it into ⚙️ DLC and DLC, but the new count (4047) still not close to the DLC count on your library (2958). It difficult to stay up-to-date with games; with DLC, this may not be feasible. 

- ⚙️ and ⎚ are not separate sets 
You're right. It doesn't seem useful to include Steam banned with Steam limited. I excluded Steam banned when counting ⚙️. The overlap looked to be 516 games for your library.

- `Steam games` is the Games Owned count from Steam profile. Therefore, the most applicable link would be https://steamcommunity.com/profiles/76561197963550511 
![image](https://user-images.githubusercontent.com/28576132/107301600-c5e1d280-6a49-11eb-83c4-e1b6470c1843.png)
The closest Barter link to that would be https://barter.vg/u/427/l/f/?tag=531

**Comment by bartervg** [comment]
I believe this was solved in an earlier layout change.
